pkg/registry/registry.go

```diff
@@ -160,26 +164,26 @@ func (r *Registry) readyHandler(rw mux.ResponseWriter, req *http.Request) {
 	}
 }
 
-func (r *Registry) registryHandler(rw mux.ResponseWriter, req *http.Request) {
+func (r *Registry) registryHandler(rw mux.ResponseWriter, req *http.Request) string {
 	// Quickly return 200 for /v2 to indicate that registry supports v2.
 	if path.Clean(req.URL.Path) == "/v2" {
-		return
+		return ""
```

Comment:
Isn't this `version` or something? It is a handler, after all.

Reply:
Yes you are probably right about this.